.github/workflows/data/test_postgres.py

```diff
@@ -0,0 +1,17 @@
+from test_sqlite import *  # NOQA
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "USER": "user",
+        "NAME": "django",
+        "PASSWORD": "postgres",
+        "HOST": "localhost",
+        "PORT": 5432,
+    },
+    "other": {
+        "ENGINE": "django.db.backends.postgresql",
+        "USER": "user",
+        "NAME": "django2",
+    },
+}
```

Comment:
I'd change an extension to don't suggest that it's ready to use where it's , e.g. `test_sqlite` is not importable here. Maybe `test_postgres.py.tpl` or `_test_postgres.py.tpl`.

Reply:
Sorry for being picky.